app/models/anime.rb

```diff
@@ -111,10 +111,7 @@ def season_year
   alias_method :year_changed?, :season_changed?
 
   def update_unit_count_guess(guess)
-    return if episode_count
-    return if episode_count_guess && episode_count_guess > guess
-    update(episode_count_guess: guess)
-    episodes.create_defaults(episode_count_guess) if episodes.length < episode_count_guess
+    update(episode_count_guess: guess) unless episode_count
```

Comment:
Now won't it overwrite mod guesses?  The old `episode_count_guess < guess` check was preventing the number from going down automatically (could only increase automatically)

Reply:
indeed 😞